README.md

```diff
@@ -140,6 +141,22 @@ In Trace Analytics you can visualize the following tables.
 
 In Insights you can see the summary table for Insights. Clicking the InsightId will take you to AWS console.
 
+### Service map
+
+[TODO: image of the map in grafana]
+
+Service map shows the same data as a service map inside X-ray console. It needs Grafana v7.4+ to use it's Node graph visualization otherwise it will show the data in a simple table.
+
+You can pan and zoom the view with buttons or you mouse. For details about the visualization see the Grafana docs [TODO: link].
+
+[TODO: image of a service with open context menu]
+
+Similar to X-ray root nodes, usually representing the client application are on the left. Each service in the map is represented as a circle and inside shows average time per transaction and transactions per minute. Color circle around the service also match colors and meaning from X-ray console representing percentage of requests, green being success, red for faults, yellow for errors and purple for throttled responses.
+
+Clicking on the service or the edge will show a context menu with links additional links for quick navigation to other relevant information from X-ray. You can for example quickly navigate to list of all error traces for a particular service and then continue to specific trace.
+
+For more details about the Service map see the X-ray documentation [TODO: link].
```

Comment:
Isn't this the X-ray documentation? Do you mean the Node graph panel documentation?

Reply:
No, I meant the aws x-ray documentation. They don't seem to have specific section for the Service map but they talk a bit about how to use it in context of other features here https://docs.aws.amazon.com/xray/latest/devguide/xray-console-insights.html for example.

Reply:
In that case, let's clarify the text.
```suggestion
For more information about the Service map, refer to the official [AWS X-ray documentation](https://docs.aws.amazon.com/xray/latest/devguide/xray-console-insights.html).
```